api/src/utils/media/csv/csv-file.ts

```diff
@@ -171,6 +202,30 @@ export class XLSXCSV implements IWorkbook {
     const rawWorkbook = xlsx.read(this.rawFile.buffer, { ...options });
 
     this.workbook = new CSVWorkBook(rawWorkbook);
+
+    this._init();
+  }
+
+  _init() {
+    switch (this.rawFile.name) {
+      case XLSX_CLASS.SAMPLE_STATION_INFORMATION:
+        this.workbook.worksheets['Sample Station Information'] = new CSVWorksheet(
```

Comment:
I think you had an enum for these worksheet names: `Sample Station Information`, etc

Reply:
Eventually we'll have to fetch those from a config somewhere (since not all templates will have the same sheet names) alongside the validation rules, etc.